pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -187,14 +187,22 @@ func isSelfNodeClientCert(csr *capi.CertificateSigningRequest, x509cr *x509.Cert
 
 var kubeletServerUsages = []capi.KeyUsage{
 	capi.UsageKeyEncipherment,
-	capi.UsageDigitalSignature,
+	capi.UsageSigning,
 	capi.UsageServerAuth,
 }
 
 func isSelfNodeServerCert(csr *capi.CertificateSigningRequest, x509cr *x509.CertificateRequest) bool {
+	if !reflect.DeepEqual([]string{"system:nodes"}, x509cr.Subject.Organization) {
```

Comment:
It's missing a check like https://github.com/kubernetes/kubernetes/pull/46884/files#diff-05476437bf221fbe4cab028bf7ffbb74R172